pkgs/applications/science/misc/root/default.nix

```diff
@@ -35,6 +66,5 @@ stdenv.mkDerivation rec {
   meta = {
     homepage = "https://root.cern.ch/";
     description = "A data analysis framework";
-    platforms = stdenv.lib.platforms.linux;
```

Comment:
If you remove the `platforms` attribute, then hydra will no longer build this package.  Can you please just change it to `platforms.unix` instead?